api4/apitestlib.go

```diff
@@ -609,7 +609,7 @@ func (th *TestHelper) CreateDmChannel(user *model.User) *model.Channel {
 	var err *model.AppError
 	var channel *model.Channel
 	if channel, err = th.App.GetOrCreateDirectChannel(th.BasicUser.Id, user.Id); err != nil {
-		mlog.Error(err.Error())
+		mlog.Critical(err.Error())
 
 		time.Sleep(time.Second)
```

Comment:
These are all test code. I would just say to remove the `mlog.Error` and `time.Sleep` and to just let it panic.

Reply:
Fair enough 👍